seb_openedx/dashboard/urls.py

```diff
@@ -0,0 +1,8 @@
+""" urls for dashboard """
+from django.conf.urls import url
+from django.contrib.admin.views.decorators import staff_member_required
+from seb_openedx.dashboard.views import TableView
+
+urlpatterns = [  # pylint: disable=invalid-name
+    url(r'^$', staff_member_required(TableView.as_view())),
```

Comment:
This means the global is_staff flag on the django user, right?
Eventually we might want to do course_staff, but for the time being, I think this is good

Reply:
Understood.